andela/travel_tool_front
# #160309166 Fix UI table and remove owner from requests table

#### What does this PR do?
Fixes the UI of the table

#### Description of Task to be completed?
- Add bottom margin to the pagination component
- Add padding to the badge for open requests/approvals count
- Remove the owner field from the requests table

#### How should this be manually tested?
- Pull this branch
- Change into the project directory `cd travel_tool_front`
- Start the server using `make start`
- Start the backend using `make start`
- Open your browser and navigate to `http://travel-tool.andela.com:3000/requests`
- Add more than 10 requests from the backend, you should see the open requests count padded
- You should also see the pagination container has a bottom margin

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160309166](https://www.pivotaltracker.com/story/show/160309166)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-06 at 6 19 30 pm" src="https://user-images.githubusercontent.com/25195857/45173958-88861480-b201-11e8-8912-bcf95e92b4b3.png">


#### Questions:
None
